.travis.yml

```diff
@@ -1,3 +1,4 @@
 import:
   - source: QubesOS/qubes-continuous-integration:R4.1/travis-dom0-r4.1.yml
   - source: QubesOS/qubes-continuous-integration:R4.1/travis-vms-r4.1.yml
+  - source: QubesOS/qubes-continuous-integration:R4.1/travis-base-r4.1.yml
```

Comment:
Only a polish style comment: we place `base` at top as it's mostly the whole definition of travis jobs. `vms` and `dom0` are env jobs.

Reply:
Fixed in the last commit.